src/e2e/java/teammates/e2e/pageobjects/Browser.java

```diff
@@ -126,10 +127,19 @@ private WebDriver createWebDriver() {
             }
             System.setProperty("webdriver.gecko.driver", TestProperties.GECKODRIVER_PATH);
 
+            FirefoxProfile profile;
+            if (TestProperties.isDevServer()) {
+                profile = new FirefoxProfile();
+            } else {
+                // Get user data from browser to bypass google blocking automated log in.
+                // Log in manually to teammates to use that log in data for e2e tests.
+                ProfilesIni profileIni = new ProfilesIni();
+                profile = profileIni.getProfile(TestProperties.FIREFOX_PROFILE_NAME);
```

Comment:
I think we should do check the existing of the profile here. It's more user friendly.